MinimedKit/PumpEvents/ChangeTimePumpEvent.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-public class ChangeTimePumpEvent: PumpEvent {
+public class ChangeTimePumpEvent: TimestampedPumpEvent {
```

Comment:
I believe so, yeah, that and the midnight events (which only include YMD components)